tests/test_views.py

```diff
@@ -116,5 +122,49 @@ def test_response_404_when_none_specified(self):
             content = json.loads(response.content.decode('utf-8'))
             self.assertCountEqual({'message': 'No checks found', 'error': 404}, content)
 
+    @override_settings(WATCHMAN_LOGIN=False)
+    def test_login_not_required(self):
+        # Reload settings - and all dependent modules - from scratch
+        reload(sys.modules['watchman.settings'])
+        reload(sys.modules['watchman.decorators'])
+        reload(sys.modules['watchman.views'])
```

Comment:
I'll pull the `reload()`s out into their own functions then.

I'm not sure what we gain by testing the `@login_required` decorator on different endpoints. The three states that test the decorator are:

1. `WATCHMAN_LOGIN = False`, bypass the `@login_required` check entirely for unauthenticated users
2. `WATCHMAN_LOGIN = True`, which for `AnonymousUser` should HTTP redirect to the site's login page
3. `WATCHMAN_LOGIN = True`, which for authenticated users should return an `HTTP 200` status and the normal status JSON

If we know the decorator works the way we want it to (eg: all three settings states are tested) on one endpoint, testing the decorator on an additional endpoint doesn't serve to test the decorator logic any more than is already done. We don't gain any more information about the decorator logic, and we have duplicated a part of our testing burden.

However, once #31 is merged in, we can switch these tests over to use that endpoint instead.

Your thoughts?